### DISCUSSION TITLE
How to add OpenAPI swagger header description?

### DISCUSSION THREAD

**Discussion by stdlout** [poem-web/poem#288] [Q&A] [answered]
Hi, I can't figure out how to add a description for a header that is set as part of a response.  Is there a property of the oai macro where the header is declared, eg:
```
enum LoginResponse {   
    #[oai(status = 200)]
    Ok(Json<Claims>, #[oai(header = "Set-Cookie")] String),
}
```
Thanks in advance.

**Comment by jonathanblade** [comment]
You are looking for [it](https://github.com/poem-web/poem/issues/269)?

**Comment by sunli829** [comment, marked as answer]
Do it like this:

https://github.com/poem-web/poem/blob/3bd9ee79e94b3f8a088a21e16648e7be6eed471c/poem-openapi/tests/response.rs#L104

**Comment by stdlout** [comment]
D'oh! That makes sense I should have tried that.  Thanks for your help!
